Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.12-alpine AS build-env
+RUN apk --no-cache add build-base git
+ADD . /src
+RUN cd /src && go build -o goapp
+
+# final stage
+FROM golang:1.12-alpine
```

Comment:
Can use just raw [alpine image](https://hub.docker.com/_/alpine). Don't need golang for final build

Reply:
Good point. Fixed.